general/g.region/printwindow.c

```diff
@@ -99,14 +112,20 @@ void print_window(struct Cell_head *window, int print_flag, int flat_flag)
         if (!prj)
             prj = "** unknown **";
 
-        if (print_flag & PRINT_SH) {
+        switch (format) {
+        case SHELL:
             fprintf(stdout, "projection=%d%s", window->proj, sep);
             fprintf(stdout, "zone=%d%s", window->zone, sep);
-        }
-        else {
+            break;
+        case PLAIN:
             fprintf(stdout, "%-*s %d (%s)\n", width,
                     "projection:", window->proj, prj);
             fprintf(stdout, "%-*s %d\n", width, "zone:", window->zone);
+            break;
+        case JSON:
+            json_object_set_number(root_object, "projection", window->proj);
```

Comment:
Let's have the projection key match the output from `PLAIN` instead of `SHELL`. 

> Current output
`"projection": 99`

> Desired output:
`"projection": "99 (Lambert Conformal Conic)"`

Reply:
Fixed